Rubberduck.Core/App.cs

```diff
@@ -155,7 +154,7 @@ private void LoadConfig()
             catch (CultureNotFoundException exception)
             {
                 Logger.Error(exception, "Error Setting Culture for Rubberduck");
-                _messageBox.Show(exception.Message, "Rubberduck", MessageBoxButtons.OK, MessageBoxIcon.Error);
+                _messageBox.NotifyError(exception.Message, "Rubberduck");
```

Comment:
I am a little ambivalent about the `Caption` property - in other examples, we seem to be passing hard-coded literal `"Rubberduck"` - seems to me that it should be made the default, or substituted with a resource key so that we maintain the default caption in only one place. OTOH, having same captions that says nothing more than just the name of program seems to be a waste of the titlebar? I'd have preferred an explicit summary of the choice (titles like `Confirm required`, `Data missing`, that kind of stuff).

Reply:
In this case I'd not want to make this into a resource string, since that's where stuff is blowing up when we couldn't set the culture for resources :/ As such a resource key doesn't make terribly much sense here. I will add a comment to that effect, though.